app/views/organisers/index.html.slim

```diff
@@ -5,6 +5,13 @@
       p
       p
       p
+      = form_tag organisers_path, :method => :post do
```

Comment:
Also i have created a organiser_token field in user. But i think we need an extra filed like organiser_token_created_at. And then apply a validation in User Model. delete it if token is 30days old. As we can have long list of organisers, with just token. 
Also, there has to be pending_approval which for all those who have yet not filled the form or clicked the link.